cmds/core/netcat/listen.go

```diff
@@ -24,6 +24,24 @@ import (
 var osListeners = map[netcat.SocketType]func(string, string) (net.Listener, error){}
 
 func (c *cmd) listenMode(output io.Writer, network, address string) error {
+	if network == "tcp" || network == "udp" {
+		network4 := network + "4"
+		err4 := c.listen(output, network4, address)
+		if err4 == nil {
+			return nil
+		}
+		network6 := network + "6"
+		err6 := c.listen(output, network6, address)
+		if err6 == nil {
+			return nil
+		}
+		return fmt.Errorf("listen IPv4: %w\nlisten IPv6: %wgi", err4, err6)
```

Comment:
I don't think you should embed newlines in an error message. You could use a single error variable and errors.Join I think.

Reply:
also, does the err have information such that you don't need the "listen IPV4"?
I suspect so.
You might just need fmt.Errorf("listenMode:%w", and you could have a single err and  errors.Join

Reply:
Nice, errors.Join seem decent!